torch/_dynamo/variables/builder.py

```diff
@@ -1820,18 +1827,15 @@ def wrap_to_fake_tensor_and_record(e, tx, *, source: Optional[Source], is_tensor
             for attr in attrs:
                 fake_inner = getattr(fake_e, attr)
                 inner = getattr(e, attr)
-                tracking_info.append(
-                    (
-                        fake_inner,
-                        inner,
-                        AttrSource(source, attr),
-                        symbolic_context.inner_contexts[attr],
-                    )
+                inner_source = AttrSource(source, attr)
+                wrap_to_fake_tensor_and_record(
+                    inner,
+                    tx,
+                    source=inner_source,
+                    is_tensor=isinstance(fake_inner, torch.Tensor),
+                    parent_context=symbolic_context,
                 )
 
-                # no need to fake-ify the inner tensors again later on
-                tx.fake_mode.fake_tensor_converter.set_tensor_memo(inner, fake_inner)
-
         for fake, real, source, symbolic_context in tracking_info:
```

Comment:
nit: now that `tracking_info` is static above, we can simplify by removing it -> the iteration here becomes unnecessary